appinventor/codelabs/talk-to-me/index.html

```diff
@@ -0,0 +1,163 @@
+  <link rel="icon" href="../../elements/favicon.png">
+  <style>
+    body {
+      font-family: "Roboto",sans-serif;
+    }
+  </style>
+
+</head>
+<body unresolved class="fullbleed">
+
+  <google-codelab title="Talk To Me Codelab" feedback-link="https://goo.gl/forms/9zDi1V3arg167lIY2">
+
+      <google-codelab-step label="Overview">
+        <p>In this codelab, you&#39;ll learn how to make your first app!
+        <h3 class="checklist">What you&#39;ll learn</h3>
+        <ul class="checklist">
+          <li>Use the <b>Designer View</b> to create the user interface.</li>
+          <li>Use the <b>Blocks View</b> to add functionality to the app.</li>
+          <li>Use the <b>Button</b> and <b>TextToSpeech</b> components.</li>
+          <li>Use <b>Event Handlers</b>.</li>
+        </ul>
+
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Create a new project">
+        <p>Create a <a href="http://ai2.appinventor.mit.edu/" target="_blank">new project</a> called
+          <code>TalkToMe</code> (no spaces).</p>
+        <img src="img/new_project.png">
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Add a Button">
+        <p>The <b>Designer View</b> shows you what the app looks like to the user.</p>
+        <p>In the <b>Palette</b> section, under the <b>User Interface</b> drawer and drag a <b>Button</b> component onto the viewer.</p>
+        <img src="img/button.png">
+        <p>Next, in the <b>Media</b> drawer, drag a <b>TextToSpeech</b> component onto the viewer.</p>
+        <p>Notice, it appears in the <b>Non-visible components</b> section under the viewer.</p>
+        <h2>Challenges</h2>
+        <ol>
+          <li>Change the text of the Button to say "Talk To Me". <i>Hint: Use the Properties section of the Button component.</i></li>
+        </ol>
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Checkoff #1">
+        <aside class="special">
+          <h2>talk-to-me-1</h2>
+          <p><b>To receive this checkoff, you must</b></p>
+          <ul class="checklist">
+            <li>Have a button that says "Talk To Me"</li>
+            <li>Describe what the Properties section of each component does</li>
+            <li>Explain why the TextToSpeech component does not appear on the viewer screen</li>
+          </ul>
+        </aside>
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Connect your phone">
+        <p>At the top of the App Inventor website, find <b>Connect</b>, and select <b>AI Companion</b> from the drop down menu.</p>
+        <img src="img/ai_companion.png">
+        <p>Open the AI2 Companion App on your phone. Select <b>scan QR code</b> and use your phone to scan the code that appears on your computer. Click <b>connect with code</b>.</p>
+        <img style="max-height: 200px" src="img/companion_app.png">
+        <p>Do you see your app on your phone? App Inventor lets you test your app on your phone as you build it on the computer!</p>
+
```

Comment:
Add troubleshooting steps:
 - If your phone has trouble loading your app, make sure your computer and your phone are using the same Wifi.
 - If your phone stops reflecting changes you make from your computer, click on Connect > Reset connection, and follow the instructions from the beginning of this step.